openwisp_notifications/templates/emails/batch_email.txt

```diff
@@ -0,0 +1,15 @@
+Summary of {{ notifications_count }} Notifications from {{ site_name }}
```

Comment:
Flag all strings as translatable using `translate` and `blocktranslate`: https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#internationalization-in-template-code